docs/hosting/scaling/execution-modes-processes.md

```diff
@@ -1,38 +1,38 @@
 # Execution modes and processes
 
-There are several different modes in which you can configure n8n to operate when executing workflows. These settings become more important as you increase the number, and complexity, of your automation workflows. Ensuring you set n8n to run in the most beneficial mode is also crucial when you begin to [scale n8n](/hosting/scaling/) to handle very large workloads.
+There are several different modes in which you can configure n8n to operate when executing workflows. These settings become more important as you increase the number, and complexity, of your automation workflows. Ensuring you set n8n to run in the most beneficial mode is also crucial when you begin to scale n8n to handle large workloads.
 
-The mode in which you n8n instance will operate is set via the following two [environment variables](/hosting/environment-variables/environment-variables/#executions):
+Set the n8n mode using the following [environment variables](/hosting/environment-variables/environment-variables/#executions):
 
-| Variable | Options | Description |
-| -------- | ------- | ----------- |
-| *EXECUTIONS_PROCESS* | `own` (default), `main`  | Determines whether each execution runs in its own process, or all run in the  main process. |
-| *EXECUTIONS_MODE* | `regular` (default), `queue` | Determines whether executions are processed by a single instance or are distributed across worker instances. |
+| Variable | Options | Default | Description |
+| :------- | :------ | :------ | :--------- |
+| `EXECUTIONS_PROCESS` | Enum string: `main`, `own` | `own` | Whether n8n executions run in their own process or the main process. |
+| `EXECUTIONS_MODE` | Enum string: `regular`, `queue` | `regular` | Whether executions should run directly or using queue. |
```

Comment:
One thing to note is that when `EXECUTIONS_MODE` is set to queue, the `EXECUTIONS_PROCESS` is ignored. n8n will behave as if you're using `main` in the worker instances.